### PR TITLE
Fix light naming

### PR DESCRIPTION
**Pull request by DMotorygin**
JIRA TICKET: RPRMAYA-1865

PURPOSE: Currently when lights are created from Hypershade (or by command line) they show wrong name in Outliner.

EFFECT FOR THE USER: Lights have the same and correct names no matter how they are created

NOTES FOR REVIEWERS: This PR also fixes the issue with lights categorized as both "lights" and "locator" in viewport and removes message about model update related to light that is spammed into the log.

### CODE REVIEW

File: FireRender.Maya.Src/FireRenderIBL.cpp [hunk 1]
```diff
@@ -241,6 +241,16 @@ void FireRenderIBL::postConstructor()
 	MObject mobj = thisMObject();
 	m_attributeChangedCallback = MNodeMessage::addAttributeChangedCallback(mobj, FireRenderIBL::onAttributeChanged, this, &status);
 	assert(status == MStatus::kSuccess);
+
+	// rename node
+	MFnDependencyNode nodeFn(thisMObject());
+	nodeFn.setName("RPRIBLShape");
+
+	MFnDagNode dagNode(thisMObject());
+	MObject parent = dagNode.parent(0, &status);
```
Comment (s1lentssh) [inline FireRender.Maya.Src/FireRenderIBL.cpp:250]: Should we check status here?

File: FireRender.Maya.Src/FireRenderIBL.cpp [hunk 2]
```diff
@@ -241,6 +241,16 @@ void FireRenderIBL::postConstructor()
 	MObject mobj = thisMObject();
 	m_attributeChangedCallback = MNodeMessage::addAttributeChangedCallback(mobj, FireRenderIBL::onAttributeChanged, this, &status);
 	assert(status == MStatus::kSuccess);
+
+	// rename node
+	MFnDependencyNode nodeFn(thisMObject());
+	nodeFn.setName("RPRIBLShape");
```
Comment (s1lentssh) [inline FireRender.Maya.Src/FireRenderIBL.cpp:247]: Is there could be only one IBL Shape with that name on the scene?

File: FireRender.Maya.Src/FireRenderSkyLocator.cpp [hunk 3]
```diff
@@ -255,6 +255,16 @@ void FireRenderSkyLocator::postConstructor()
 	MObject mobj = thisMObject();
 	m_attributeChangedCallback = MNodeMessage::addAttributeChangedCallback(mobj, FireRenderSkyLocator::onAttributeChanged, this, &status);
 	assert(status == MStatus::kSuccess);
+
+	// rename
+	MFnDependencyNode nodeFn(thisMObject());
+	nodeFn.setName("RPRSkyShape");
+
+	MFnDagNode dagNode(thisMObject());
+	MObject parent = dagNode.parent(0, &status);
```
Comment (s1lentssh) [inline FireRender.Maya.Src/FireRenderSkyLocator.cpp:264]: Should we check status here?

File: FireRender.Maya.Src/Lights/FireRenderLightCommon.h [hunk 4]
```diff
@@ -24,6 +24,7 @@ class FireRenderLightCommon : public MPxLocatorNode
 	virtual ~FireRenderLightCommon();
 
 	virtual void postConstructor() override;
+	virtual bool  excludeAsLocator() const override;
```
Comment (s1lentssh) [inline FireRender.Maya.Src/Lights/FireRenderLightCommon.h:27]: Please remove spaces

File: FireRender.Maya.Src/Lights/IES/FireRenderIESLight.cpp [hunk 5]
```diff
@@ -246,6 +246,24 @@ void* FireRenderIESLightLocator::creator()
...
+
+	MStatus status;
+	MObject mobj = thisMObject();
+
+	// rename node
+	MFnDependencyNode nodeFn(thisMObject());
+	nodeFn.setName("RPRIESShape#");
+
+	MFnDagNode dagNode(thisMObject());
+	MObject parent = dagNode.parent(0, &status);
```
Comment (s1lentssh) [inline FireRender.Maya.Src/Lights/IES/FireRenderIESLight.cpp:261]: Should we check status here?

File: FireRender.Maya.Src/Lights/PhysicalLight/FireRenderPhysicalLightLocator.cpp [hunk 6]
```diff
@@ -56,6 +56,16 @@ void FireRenderPhysicalLightLocator::postConstructor()
 	
 	m_attributeChangedCallback = MNodeMessage::addAttributeChangedCallback(mobj, FireRenderPhysicalLightLocator::onAttributeChanged, this, &status);
 	assert(status == MStatus::kSuccess);
+
+	// rename node
+	MFnDependencyNode nodeFn(thisMObject());
+	nodeFn.setName("RPRPhysicalLightShape#");
+
+	MFnDagNode dagNode(thisMObject());
+	MObject parent = dagNode.parent(0, &status);
```
Comment (s1lentssh) [inline FireRender.Maya.Src/Lights/PhysicalLight/FireRenderPhysicalLightLocator.cpp:65]: Should we check status here?

File: FireRender.Maya.Src/FireRenderIBL.cpp (hunk 2 above)
Comment (DMotorygin) [reply FireRender.Maya.Src/FireRenderIBL.cpp:247]: There could be only one RPRIBL object on the scene

**Pull request merged**